From 231a054d94ee9eced6e116829d5a9f7706d7bb41 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Andreas=20M=C3=BCller?= <schnitzeltony@googlemail.com>
Date: Tue, 27 Oct 2015 01:16:47 +0100
Subject: [PATCH] Makefile.in: avoid building target cross-toolset
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Upstream-Status: Inappropriate [cross compiling specific]

Signed-off-by: Andreas Müller <schnitzeltony@googlemail.com>
---
 Makefile.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile.in b/Makefile.in
index 61b16ab..37405f2 100644
--- a/Makefile.in
+++ b/Makefile.in
@@ -242,7 +242,7 @@ bootstrap: compilerplugins
 # Note: this will pipe through all gbuild targets to ... gbuild
 #       with some translations like "build"->"all" for historic reasons
 #
-build: bootstrap fetch $(if $(CROSS_COMPILING),cross-toolset)
+build: bootstrap fetch
 ifneq ($(filter-out WNT MACOSX IOS,$(OS)),)
 	mkdir -p $(INSTDIR)
 	$(SRCDIR)/solenv/bin/install-gdb-printers -a $(INSTDIR) -c
-- 
2.1.0

