From 22ae425b34775feb015eae7bea113ce15f3cce4d Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Andreas=20M=C3=BCller?= <schnitzeltony@googlemail.com>
Date: Mon, 15 Jul 2013 17:13:33 +0200
Subject: [PATCH] configure.ac: remove special cross code for Pyuno
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Upstream-Status: Inappropriate [configuration specific]

Signed-off-by: Andreas Müller <schnitzeltony@googlemail.com>
---
 configure.ac |   33 +++------------------------------
 1 files changed, 3 insertions(+), 30 deletions(-)

diff --git a/configure.ac b/configure.ac
index 0818adf..8d63c3c 100644
--- a/configure.ac
+++ b/configure.ac
@@ -7380,7 +7380,7 @@ if test $enable_python = system; then
         # Fallback: Accept these in the environment, or as set above
         # for MacOSX.
         :
-    elif test "$cross_compiling" != yes; then
+    else
         # Unset variables set by the above AM_PATH_PYTHON so that
         # we actually do check anew.
         unset PYTHON am_cv_pathless_PYTHON ac_cv_path_PYTHON am_cv_python_version am_cv_python_platform am_cv_python_pythondir am_cv_python_pyexecdir
@@ -7400,33 +7400,6 @@ if test $enable_python = system; then
             PYTHON_CFLAGS="-I$python_include"
             PYTHON_LIBS="-L$python_libdir -lpython$python_version $python_libs"
         fi
-    else
-        dnl How to find out the cross-compilation Python installation path?
-        dnl Let's hardocode what we know for different distributions for now...
-        for python_version in 2.6; do
-            if test -f ${MINGW_SYSROOT}/include/python${python_version}/Python.h; then
-                PYTHON_CFLAGS="-I${MINGW_SYSROOT}/include/python$python_version"
-                PYTHON_LIBS="-L${MINGW_SYSROOT}/lib -lpython$python_version $python_libs"
-                AC_MSG_CHECKING([for python.exe])
-                libo_MINGW_CHECK_DLL([libpython$python_version])
-                libo_MINGW_CHECK_DLL([libreadline6])
-                libo_MINGW_CHECK_DLL([libtermcap])
-                # could we somehow extract the really mingw python version from
-                # actual distro package?
-                # 2.6.2 currently on OpenSUSE 12.1?
-                # rpm -q mingw32-python => mingw32-python-2.6.2-17.17.noarch
-                PYTHON_VERSION=$python_version.2
-                PYTHON_VERSION_MAJOR=`echo $python_version | cut -d . -f 1`
-                PYTHON_VERSION_MINOR=`echo $python_version | cut -d . -f 2`
-                break
-            fi
-        done
-        AC_MSG_CHECKING([for python version])
-        AS_IF([test -n "$PYTHON_VERSION"],
-              [AC_MSG_RESULT([$PYTHON_VERSION])],
-              [AC_MSG_RESULT([not found])
-               AC_MSG_ERROR([no usable python found])])
-        test -n "$PYTHON_CFLAGS" && break
     fi
     # let the PYTHON_FOR_BUILD match the same python installation that
     # provides PYTHON_CFLAGS/PYTHON_LDFLAGS for pyuno, which should be
@@ -7439,7 +7412,7 @@ case $enable_python in
 system)
     SYSTEM_PYTHON=YES
 
-    if test "$CROSS_COMPILING" != YES; then
+#    if test "$CROSS_COMPILING" != YES; then
         dnl Check if the headers really work
         save_CPPFLAGS="$CPPFLAGS"
         CPPFLAGS="$CPPFLAGS $PYTHON_CFLAGS"
@@ -7464,7 +7437,7 @@ int main(int argc, char **argv) {
         AC_LANG_POP(C)
 
         dnl FIXME Check if the Python library can be linked with, too?
-    fi
+#    fi
     ;;
 
 internal)
-- 
1.7.6.5

