From b9165f44e3e1d0b07240c1b6d7dae9a835e96ca6 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Andreas=20M=C3=BCller?= <schnitzeltony@googlemail.com>
Date: Wed, 21 Oct 2015 23:27:01 +0200
Subject: [PATCH] configure.ac: skip cross-compile section  - ist does not work
 for oe-environment
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Upstream-Status: Inappropriate [configuration]

Signed-off-by: Andreas Müller <schnitzeltony@googlemail.com>
---
 configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index f72d921..0f75d87 100644
--- a/configure.ac
+++ b/configure.ac
@@ -4465,7 +4465,7 @@ dnl machine when doing cross-compilation
 dnl ===================================================================
 
 m4_pattern_allow([PKG_CONFIG_FOR_BUILD])
-if test "$cross_compiling" = "yes"; then
+if test "$cross_compiling" = "yesfoo"; then
     AC_MSG_CHECKING([for BUILD platform configuration])
     echo
     rm -rf CONF-FOR-BUILD config_build.mk
-- 
2.1.0

